addons/nuxeo-retention/nuxeo-retention-core/src/main/resources/OSGI-INF/retention-pageproviders.xml

```diff
@@ -100,8 +101,8 @@
         <quickFilter name="hasNotLegalHold">
           <clause>ecm:hasLegalHold = 0</clause>
         </quickFilter>
-        <quickFilter name="exceptTrashed">
-          <clause>ecm:isTrashed = 0</clause>
+        <quickFilter name="includeTrashed">
+          <clause>ecm:isTrashed = 1</clause>
```

Comment:
With `ecm:isTrashed = 0` in the fixed part we'll end up with `ecm:isTrashed = 0 AND ecm:isTrashed = 1` => no results, no?

Reply:
Indeed! So maybe adding a quickfilter is enough, and we do not modify the fixed part of the query.

Reply:
Initial requirement as described in the ticket was "Default retention search shouldn't list trashed documents" which this doesn't do and we already had a quickfilter to exclude trashed documents so effectively you're adding a new quick filter to list trashed documents under retention (which I thought you shouldn't be able to have in the first place, no?)

Reply:
You can have documents under retention located in the trash (if compliance mode is not explicitly activated). So the original requirement is to get non trashed documents in the listing. But, as you noted correctly, if we have a fixed part excluding trashed document, we wouldn't be able to make any operation on these documents (extending retention period / apply a legal hold / attach a retention rule). So we would be loosing a default feature (even if we agree it is strange to perform retention actions on trashed documents)

Reply:
But my point is we already have a quick filter to show only non trashed documents.... is there a new requirement to show only trashed now ?

Reply:
Not really, so well considered, we can keep the original  code. The idea would be to get the quickfilter `exceptTrashed` selected by default.